src/ucp/tag/offload.c

```diff
@@ -602,8 +613,12 @@ ucs_status_t ucp_tag_offload_rndv_zcopy(uct_pending_req_t *self)
                                     sizeof(rndv_hdr), iov, iovcnt, 0,
                                     &req->send.state.uct_comp);
     if (UCS_PTR_IS_ERR(rndv_op)) {
-        return UCS_PTR_STATUS(rndv_op);
+        status = UCS_PTR_STATUS(rndv_op);
+        ucs_assert(status != UCS_ERR_NO_RESOURCE);
```

Comment:
why?

Reply:
yes, `UCS_ERR_NO_RESOURCE` is a possible return value - fixed